services/core/MasterDriverAgent/master_driver/interfaces/modbus.py

```diff
@@ -70,7 +71,7 @@ def modbus_client(address, port):
 MODBUS_READ_MAX = 100
 PYMODBUS_REGISTER_STRUCT = struct.Struct('>H')
 
-path = os.path.dirname(os.path.abspath(__file__))
+path = os.path.join(utils.get_home(), "examples", "configurations", "drivers")
```

Comment:
I believe this should be using 

from volttron.platform import get_volttron_root

Rather than get_home...get_home will give you the volttron home, which will not have examples of driver configurations.